openlibrary/coverstore/db.py

```diff
@@ -144,6 +144,41 @@ def delete(id):
         t.commit()
 
 
+def print_author_query(query):
+    # create the process to print for the query
+    process = Process()
```

Comment:
This would need `from subprocess import Process` or maybe even more modern, `from subprocess import run`.